### DISCUSSION TITLE
24 mars : Tâches accomplies + à faire pour 29 mars

### DISCUSSION THREAD

**Discussion by blanchette31** [BIO6032/2022-SEIR-reproduction#20] [General]
**Ce qui a été accompli dans la séance d'aujourd'hui :** 

- création branche "model" pour l'implémentation du modèle
-  déplacé les folders "figures" et "data" dans le folder "code"
- ajout de l'environnement de travail Julia
- ajout du fichier script du modèle : "SEIR_model.jl"
- ajout des packages nécessaires
- script écrit jusqu'à la description du problème
- décision sur la structure du script (code pour les simulations toutes ensembles et code pour la visualisation nécessaire pour le figures séparés étant donné que les figures sont majoritairement de calculs effectués à partir des mêmes simulations

**À faire d'ici mardi le 29**
- @Marine1719 partager l'article qui pourrait avoir les données / valeurs des paramètres soit dans cette discussion ou dans une nouvelle
- voir dans l'article de Brown pour les valeurs des paramètres
- ...? 

**Choses à avancer**
- README.md objectifs (correction/vérification)
- README.md définition du problème
- README.md définitions et jargon 
- review les pull requests actuels et suggérer des modifications / faire d'autres commits / accepter les pull requests
- merge les différentes branches créées pour ajouter les orcid  id

**Comment by blanchette31** [comment]
R0 équivalent à (beta * N)/ fr

Donc, beta = (R0 * fr)/N
beta = R0 / (14 * 10 000)


Tiré directement de l'article : T_R = 14 jours et fr = 1/T_R donc, fr = 1/14.
fs (taux de conversion d'asymptomatique à symptomatique) = 0.6 
ft = 1/7 ou 2/7 (pour tester l'augmentation du contact tracing. 
s = 0.9 (prend en compte l'efficacité du test et les étudiants qui ratent les tests)
tspan = 100 jours
fq = ~1 approximation quasi steady-state donc on considère qu'un individu qui développe des symptômes se fait placer quasi-instantanément en quarantaine puisqu'on assume que la grande majorité de la transmission de la maladie se fait par les individus asymptomatiques et infectieux. On justifie cela par le fait que les personnes symptomatiques peuvent être rapidement détectés et placés en quarantaine. Donc, je crois que le taux fq doit être très rapide, donc près de 1 puisqu'on considère que 100% des individus symptomatiques seront placés en quarantaine. 

Initial :
N = 10 000
 S = 9999
R = 0
Q = 0 
A = 1 
I = 0
E(t) << 1  pour les figures 3 il a été estimé en prenant le nombre de cas par jours dans Suffolk County pondéré à la taille de population du campus (10 000 individus). Pour les figures 2 je ne crois pas que la valeur de E(t) varie mais je ne vois pas une valeur spécifiée pour E(t) autre que ça doit être beaucoup plus faible que 1. 

Je ne crois pas que c'est écrit explicitement mais je crois que c'est plutôt A qui commence avec 1 individu que I en raison de la phrase qui le laisse sous-entendre : "Without exogenous input—a cloistered university—an initial input of infections, A(0)> 0, would be needed to generate infections."


Voici également le lien pour les données de Suffolk County utilisées pour produire la courbe dans la figure 3B (Source 13 dans l'article partagé par Marine) : https://www.mass.gov/info-details/covid-19-response-reporting#incidental-covid-19-report-


Et voici un exemple d'archive des données de ce site (les données pour Suffolk County se retrouvent à la page 23) : https://www.mass.gov/doc/weekly-covid-19-public-health-report-september-2-2020/download

**Comment by Marine1719** [comment]
* Pour ajouter un commentaire --> Tr est le temps que dure la période d'infectiosité qui est donc de 14 jours. 
* ft est le taux auquel les individus A(t) sont testés. Par contre ici @blanchette31  où as tu trouvé la valeur 1/7, ils parlent de 0,2% de personnes testées positives par jour dans l'article mais je ne vois rien d'autres

**Reply by blanchette31** [reply]
Non ft c'est la fréquence du "surveillance testing". Mais en fait, je viens d'aller revoir et en fait sous la figure 2 il dit que la fréquence des tests utilisée pour les simulations était de 2/7 et donc deux tests par semaine. Le 0,2% dont tu parles est tenu en compte dans le paramètre "s". Au bas de l'article, dans la section matériel et méthodes, il décrit ce que "s" tient en compte et sa valeur.
